delfick_logging.py

```diff
@@ -150,7 +178,9 @@ def setup_logging(log=None, level=logging.INFO, syslog="", syslog_address="", on
     """
     log = log if log is not None else logging.getLogger(log)
 
-    if syslog:
+    if tcp_address:
```

Comment:
If tcp make sure you spit out to stderr if the connection blocks or dies. (and\or a metric)